webapp/channels/src/components/activity_and_insights/insights/insights.tsx

```diff
@@ -41,17 +40,20 @@ type SelectOption = {
 
 const Insights = () => {
     const dispatch = useDispatch();
-
-    // check if either of focalboard plugin or boards product is enabled
-    const focalboardPluginEnabled = useSelector((state: GlobalState) => state.plugins.plugins?.focalboard);
-    let focalboardProductEnabled = false;
     const products = useProducts();
+
+    let focalboardEnabled = false;
```

Comment:
Not all that important, much optional 
```suggestion
    let boardsEnabled = false;
```